SMPyBandits/Arms/Arm.py

```diff
@@ -43,6 +43,17 @@ def draw(self, t=None):
         """ Draw one random sample."""
         raise NotImplementedError("This method draw(t) has to be implemented in the class inheriting from Arm.")
 
+    def oracle_draw(self, t = None):
+        # draw the arm as usual but return the mean
```

Comment:
Nice idea.
Use a `""" doctstring"""` instead of a line `# comment` please, to document this method.

Reply:
ok